sdk/appconfiguration/Azure.Data.AppConfiguration/src/Azure.Data.AppConfiguration.csproj

```diff
@@ -1,9 +1,9 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <Description>This is the Microsoft Azure Application Configuration Service client library</Description>
     <AssemblyTitle>Microsoft Azure.Data.AppConfiguration client library</AssemblyTitle>
-    <Version>1.3.0-beta.3</Version>
+    <Version>1.2.1</Version>
```

Comment:
I'm curious about this because it's an area I don't understand.  How do we think about maintaining branches for beta vs. GA releases?  If we pull back from 1.3.x to 1.2.x, do we need to remove the APIs that were part of the beta?

Reply:
Yeah.  I'm going to cherry-pick this commit over to a hotfix branch and then reset main.